Ecommerce/index.html

```diff
@@ -8,6 +8,13 @@
 
   <body style="margin: 0; padding: 0">
     <!-- Start coding from the lines below for ECOMMERCE TASK -->
+   
     <!-- to import image from public folder, do it like this = <img src="../public/css.icon.png" alt="CSS image"> -->
+  <img src="../public/css.icon.png">
+  <ul>
```

Comment:
The closing ul tag should be indented to align with the opening ul tag for better readability.